docs/locale/ja_JP/source/fabric_model.rst

```diff
@@ -1,170 +1,151 @@
+操作および管理できます。Hyperledger Fabricのこの"許可型"概念は、チャネルの存在と機能と相まって、
+プライバシーと機密性が最も重要な関心事であるシナリオに対処するのに役立ちます。
+
+暗号化の実装をよりよく理解するには :doc:`msp` のトピックを参照してください。
+また、Hyperledger Fabricで使用される署名、検証、認証のアプローチを参照してください。
+
+合意形成
 ---------
 
-In distributed ledger technology, consensus has recently become synonymous with
-a specific algorithm, within a single function. However, consensus encompasses more
-than simply agreeing upon the order of transactions, and this differentiation is
-highlighted in Hyperledger Fabric through its fundamental role in the entire
-transaction flow, from proposal and endorsement, to ordering, validation and commitment.
-In a nutshell, consensus is defined as the full-circle verification of the correctness of
-a set of transactions comprising a block.
-
-Consensus is achieved ultimately when the order and results of a block's
-transactions have met the explicit policy criteria checks. These checks and balances
-take place during the lifecycle of a transaction, and include the usage of
-endorsement policies to dictate which specific members must endorse a certain
-transaction class, as well as system chaincodes to ensure that these policies
-are enforced and upheld.  Prior to commitment, the peers will employ these
-system chaincodes to make sure that enough endorsements are present, and that
-they were derived from the appropriate entities.  Moreover, a versioning check
-will take place during which the current state of the ledger is agreed or
-consented upon, before any blocks containing transactions are appended to the ledger.
-This final check provides protection against double spend operations and other
-threats that might compromise data integrity, and allows for functions to be
-executed against non-static variables.
-
-In addition to the multitude of endorsement, validity and versioning checks that
-take place, there are also ongoing identity verifications happening in all
-directions of the transaction flow.  Access control lists are implemented on
-hierarchical layers of the network (ordering service down to channels), and
-payloads are repeatedly signed, verified and authenticated as a transaction proposal passes
-through the different architectural components.  To conclude, consensus is not
-merely limited to the agreed upon order of a batch of transactions; rather,
-it is an overarching characterization that is achieved as a byproduct of the ongoing
-verifications that take place during a transaction's journey from proposal to
-commitment.
-
-Check out the :doc:`txflow` diagram for a visual representation
-of consensus.
+分散台帳技術では、合意形成は単一機能内における特定のアルゴリズムを表す言葉として近年では捉えられています。
+しかしながら、合意形成は単にトランザクションの順序に合意するためだけではありません。
+Hyperledger Fabricでは、提案およびエンドースメントから、オーダリング、検証およびコミットに至るまで、
+トランザクションフロー全体における基本的な役割を通じて、この合意形成の役割の違いが強調されています。
+簡単に言うと、コンセンサスは、ブロックを構成する一連のトランザクションの正確さを完全に検証する
+一連のプロセスとして定義されます。
+
+ブロックのトランザクションの順序と結果が明示的なポリシー基準のチェックを満たすことによって、
+合意形成は最終的に達成されます。これらのチェックとバランスは、トランザクションのライフサイクルの間に行われ、
```

Comment:
バランス とは何を指すのか分かりませんでした。アセットのことか?

Reply:
"checks and balances"で、(権力などの)「抑制と均衡」の意味のようです。
> a system that allows each branch of a government to amend or veto acts of another branch so as to prevent any one branch from exerting too much power
(https://www.merriam-webster.com/dictionary/checks%20and%20balances)

なので、組織がエンドースによってトランザクションの正しさを互いに検証していることを指しているような気がします。